Dockerfile

```diff
@@ -22,9 +22,16 @@ RUN apt-get update \
 
 RUN wget -q https://github.com/mikefarah/yq/releases/download/v4.14.2/yq_linux_amd64 -O /usr/bin/yq && chmod +x /usr/bin/yq
 
-RUN apt-get install -y --no-install-recommends git-lfs && \
-    git lfs install && \
-    rm -r /var/lib/apt/lists/*
+# Install latest version of git
+RUN apt-get install -y --no-install-recommends software-properties-common  && \
```

Comment:
Remove git installation from above step.

Reply:
wouldnt it be better if we freeze as opposed to getting the latest?

Reply:
Removed git step and froze version for both `git` and `git-lfs`.